### DISCUSSION TITLE
加载earthsdk时，报width must be less than or equal to the maximum texture size.Check maximumTextureSize.

### DISCUSSION THREAD

**Discussion by NSinC-LJ** [zouyaoji/vue-cesium#130] [Q&A]
![图片](https://user-images.githubusercontent.com/7565905/141440319-1521b395-dc37-43c6-a818-580c19ce252e.png)
@zouyaoji

**Comment by zouyaoji** [comment]
@NSinC-LJ 
参考这儿弄没？我看了一下这儿加的应该是火星最新的版本了，没问题的。

https://zouyaoji.top/vue-cesium/#/zh-CN/component/platforms/vc-demo-mars3d

**Reply by zouyaoji** [reply]
看错了，说的是earthsdk，我再瞅瞅

**Reply by zouyaoji** [reply]
没重现，参考这个例子哈
https://zouyaoji.top/vue-cesium/#/zh-CN/component/platforms/vc-demo-earth-sdk 
如有问题建议提 issue
